DHL/JSON/9df77954-633e-4a19-8e78-a2e73e34050f.json

```diff
@@ -0,0 +1,998 @@
+[
+  {
+    "ShpInfReqDef": {
+      "Id": " 9df77954-633e-4a19-8e78-a2e73e34050f",
+      "Nm": "Returned Export - GR",
+      "Dsc": "This import shipment is a returning commercial export",
+      "DrvdFromShpInfReqDef": {},
+      "LedOwnOrgn": {
+        "CtryCd": "GR"
+      },
+      "ApplyAllInd": false,
+      "PrsnDatSenstyClssLvl": "Low",
+      "QuickGuideEnabledInd": false,
+      "UICmpt": {
+        "CmptNm": "RetExport",
+        "DisplayTtl": "Returned Export",
+        "Tooltip": "",
+        "CmptTy": "Composed",
+        "DisplayDsc": {
+          "DisplayTxt": ""
+        },
+        "Hint": ""
+      },
+      "Ent": [
+        {
+          "Id": "8453c6a8-6fa0-494e-9a19-fcb606f2b4ee",
+          "Nm": "Returned Export entity",
+          "Dsc": "Returned Export Discriptiption ",
+          "isImg": false,
+          "isExcl": false,
+          "isMan": true,
+          "Ent": [
+            {
+              "Id": "f87d71de-374e-4b7d-8eb9-fde7fe2c9adc",
+              "Nm": "Shipment Customer",
+              "Dsc": "Shipment Customer Entity",
+              "isImg": false,
+              "isExcl": false,
+              "isMan": true,
+              "Ent": [
+                {
+                  "Id": "181beb24-9acb-4b2b-9839-3798cbc93406",
+                  "Nm": "Business Party",
+                  "Dsc": "Business Party Entity",
+                  "isImg": false,
+                  "isExcl": false,
+                  "isMan": true,
+                  "Ent": [
+                    {
+                      "Id": "206eb8bd-37df-4696-b778-7b81f3695ab7",
+                      "Nm": "Individual Customer",
+                      "Dsc": "Individual Customer Entity",
+                      "isImg": false,
+                      "isExcl": false,
+                      "isMan": true,
+                      "Ent": [
+                        {
+                          "Id": "d4c1b6ad-3a6e-442b-825d-735bacfd1bbf",
+                          "Nm": "Consent for Returned Shipment",
+                          "Dsc": "Consent for Returned Shipment Entity",
+                          "isImg": false,
+                          "isExcl": false,
+                          "isMan": true,
+                          "Ent": [
+                            {
+                              "Id": "e51fc277-a311-4724-9dba-341b2d3e8e18",
+                              "Nm": "Consent Category",
```

Comment:
nm should be relevant "Consent Returned Shipment Category"

Reply:
Done